_works/lume.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+category: work
+title: 'Lume'
+subtitle: 'Mobile App'
+client: 'Lume'
+tags: 'mobile'
+image:
+ - url: '../../public/images/work/thumbs/thumb-lume.jpg'
+links:
+ - type: website
+   link: #
+ - type: iOS
+   link: #
+ - type: Android
+   link: #
+sitemap: false
+---
+
+We've built Lume, you can use your bluetooth scale to mesure your weight and keep track of your burned calories.
```

Comment:
This text, is it suitable ?

Reply:
for now dont' incude this one, stick it in the drafts